Doc/cmus.txt

```diff
@@ -513,6 +516,10 @@ pl-rename <name>
 player-next (*b*)
 	Skips to the next track.
 
+player-next-album (*B*)
```

Comment:
I wonder if `player-next-shuffled` might be a better name for this.

Reply:
It does also work outside shuffle mode where it'll just advance to the next album in library or sorted order.

What you suggest sounds like the `player-next-rand` I implemented originally, but after playing around with it I thought this would be more intuitive and more discoverable.

Reply:
OTOH, without reading the code, I'd expect `player-next-album` to go to the next album regardless of the shuffle mode. I'd read `player-next-shuffled` as `Skips to the next shuffled entry. If not playing from the library, skips to the next track.`.

> would be more intuitive and more discoverable

I agree.

Reply:
In that case, after thinking about it, I agree that it's better than the alternatives.